cargo/crates.bzl

```diff
@@ -16,15 +16,6 @@ def _new_git_repository(name, **kwargs):
 
 def raze_fetch_remote_crates():
 
-    _new_http_archive(
-        name = "raze__ahash__0_3_2",
-        url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/ahash/ahash-0.3.2.crate",
-        type = "tar.gz",
-        strip_prefix = "ahash-0.3.2",
-
-        build_file = Label("//cargo/remote:ahash-0.3.2.BUILD"),
```

Comment:
Just to clarify: I think these `cargo/remote/<removed-dep>.BUILD` files should be removed in this PR too.

Reply:
Done.